Source/Core/Core/HW/DSPHLE/UCodes/AX.h

```diff
@@ -76,11 +76,6 @@ class AXUCode : public UCodeInterface
 protected:
   enum MailType
   {
-    MAIL_RESUME = 0xCDD10000,
-    MAIL_NEW_UCODE = 0xCDD10001,
-    MAIL_RESET = 0xCDD10002,
-    MAIL_CONTINUE = 0xCDD10003,
-
     // CPU sends 0xBABE0000 | cmdlist_size to the DSP
     MAIL_CMDLIST = 0xBABE0000,
```

Comment:
You can leave this as is but this is only used in *one* place.  Seems strange to me to have it in an enum called `MailType` all by itself.  I wonder if we could kill this enum with a function or something:

```
bool IsCmdList(u32 val)
{
   const u32 mask = 0xFFFF0000;
   const u32 expected_val = 0xBABE0000;
   return (val & mask) == expected_val;
}
```

or maybe there is something more appropriate?

Reply:
The way it's actually used is like this:

```C++
  else if ((mail & MAIL_CMDLIST_MASK) == MAIL_CMDLIST)
  {
    // A command list address is going to be sent next.
    set_next_is_cmdlist = true;
    cmdlist_size = (u16)(mail & ~MAIL_CMDLIST_MASK);
  }
```

Probably the best choice is to just use `constexpr u32` instead of `enum` (and it could also be put in the cpp file instead of the header, but I have less of an opinion on that front).